FingerprintDialogKotlin/app/src/main/java/com/example/android/fingerprintdialog/MainActivity.kt

```diff
@@ -287,6 +280,53 @@ class MainActivity : AppCompatActivity(),
         return super.onOptionsItemSelected(item)
     }
 
+    private fun instanceOfBiometricPrompt(): BiometricPrompt {
```

Comment:
nit: Not sure if `instanceOfX` is a common naming pattern. What about `createBiometricPrompt` instead?

Reply:
along the lines of what Curtis mentioned, instance might be confused with singletons etc